### ISSUE TITLE
This element is now deprecated

### ISSUE THREAD

**Issue by CTOJoe** [PolymerElements/neon-animation#238]
Under the changes in 2.0 section of the README, it states this element is now deprecated.

- Does that mean we should stop using neon-animated-pages?
- If use of neon-animated-pages is deprecated, what would be a valid replacement?

**Comment by jamrizzi** [comment]
Yes, could someone please explain why this is deprecated, as well as the implications of the depreciation.

**Comment by prateeklal** [comment]
@CTOJoe We can use web animations API or css animations as a replacement.

**Comment by gurpreetatwal** [comment]
They've just published a blog post on it: https://www.polymer-project.org/blog/2017-07-12-an-update-on-neon-animation.html
